ui/trace_window.cpp

```diff
@@ -55,9 +55,34 @@
 
 namespace
 {
-const std::vector<std::string> kAppTypes{ "Vulkan APK", "OpenXR APK", "Command Line Application" };
-const int                      kGfxrCaptureButtonId = 1;
-const int                      kPm4CaptureButtonId = 2;
+enum class AppTypes
+{
```

Comment:
@RenfengLiu Does VulkanCLI support both OpenXR and non-OpenXR vulkan app or just non-OpenXR?

sorry i just remember a feedback from the users: people got a bit confused when they see Vulkan and OpenXR Vulkan. I am thinking if it would be clearer to have following types:
{
    kVulkan_OpenXR
    kVulkan_Non_OpenXR
    kVulkanCLI_Non_OpenXR <- this depends on @RenfengLiu 's answer of the above question
    kGLES_OpenXR
}